dynamicio/mixins.py

```diff
@@ -736,8 +775,15 @@ def _write_to_postgres(self, df: pd.DataFrame):
 
         connection_string = f"postgresql://{db_user}:{db_password}@{db_host}:{db_port}/{db_name}"
 
+        if self.options.get("model"):
+            raise ValueError("`model` must not be provided")
+
```

Comment:
Redundant as per above comment. Even if we keep it in this will never be true after removing  live 719:
```python
@allow_options({"model"})
def _write_to_postgres(self, df: pd.DataFrame):
```

`model` is no longer an allowed option.